RELEASE_TODO.md

```diff
@@ -0,0 +1,10 @@
+The release procedure:
+
+1. Compile the current version of the crate with `cargo contract build --release --features "contract"`, optionally using one of dockerized build environments.
```

Comment:
Compiling with dockerized, reproducible env is a requirement - not an option. We have `ink-dev` docker image. 
```
INK_DEV_IMAGE = public.ecr.aws/p6e8q1z1/ink-dev:2.1.0

.PHONY: build-dockerized
build-dockerized: # Build contract in docker contrainter
	@docker run --rm \
		--name ink-dev \
		--volume "$(shell pwd)":/code \
		$(INK_DEV_IMAGE) \
		make build ;
	@$(eval codehash = `jq '.source .hash' "target/ink/common_point_token.json" | sed 's/"//g'`)
	@echo "Contract code hash:"
	@echo "$(codehash)"
ifdef verify
	@if [ $(verify) = $(codehash) ]; then \
		echo "\033[0;32mContract code hash match!" ; \
	else \
		echo "\033[0;31m$(verify)" ; \
		echo "Contract code hash doesn't match!" ; \
	fi
endif
```
This will work.

Reply:
If we don't use any "public" verification, it completely doesn't matter how we build

Reply:
It does matter - we will allow people to verify for themselves. Also, if there's a "public verification" in the future, we might be able to reuse it.

There's no additional cost for doing as I showed above and all the downsides.